.github/workflows/test.yml

```diff
@@ -2,41 +2,51 @@ name: test
 
 on: push
 
+env:
+  E2E_DATAVERSE_BRANCH: 9588-datasets-api-extension
```

Comment:
The problem is that the integration tests must point to the Dataverse backend branch (image tag) used by the specific version of the js-dataverse package used, otherwise they could fail, not having the necessary Dataverse API changes.

We could set develop as you suggest (seems like a cleaner solution), but then we need to make sure to merge backend PRs first. This way the frontend PRs should wait (before accepting them) for the backend PRs to be merged for the e2e action to pass.

Reply:
is there a way to pass repo and branch variables with the commit message?

Reply:
We could read the branch name of the commit message from the action via `${{ github.event.head_commit.message }}` and detect if a branch name has been sent (e.g. surrounded the branch name by some text markers or using regex). In that case, we use that tag in the action. Otherwise, we use develop. What do you think?

Reply:
It's what dataverse-ansible does (check for specified repo / branch, else assume defaults)

Reply:
Back to the order of PRs being merged... recently we merged [a frontend PR](https://github.com/IQSS/dataverse-frontend/pull/112) and [a JS PR](https://github.com/IQSS/dataverse-client-javascript/pull/60) before we merged the [backend PR](https://github.com/IQSS/dataverse/pull/9592) they depend on. This is backwards. The backend PR should be merged first. Otherwise, the frontend and js work will have to be revisited if changes are required in the backend PR. I also mentioned this here: https://iqss.slack.com/archives/C010LA04BCG/p1686690116229119?thread_ts=1686603802.872139&cid=C010LA04BCG